rest/client/client.go

```diff
@@ -87,6 +97,17 @@ type BaseResponse struct {
 	PaginationHooks
 }
 
+// PaginationHooks are links to next and/or previous pages. Embed this struct into your API response if
+// your endpoint is going to support pagination.
```

Comment:
I'm wondering about the phrasing here
Should `your endpoint` be `the endpoint` ? If I was a user to this I'm not sure if the user would consider possession of the endpoint